x-pack/plugins/alerting/server/application/rule/methods/bulk_disable/bulk_disable_rules.ts

```diff
@@ -50,7 +50,7 @@ export const bulkDisableRules = async <Params extends RuleParams>(
     throw Boom.badRequest(`Error validating bulk disable data - ${error.message}`);
   }
 
-  const { ids, filter } = options;
+  const { ids, filter, untrack = true } = options;
```

Comment:
I am wondering if we need to default to false. @shanisagiv1 what is better for our user using directly our API to untracked the alerts or to leave them alone.